docs/src/examples/components/Input/Variations/InputExampleIconClearable.shorthand.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { Input } from '@stardust-ui/react'
+
+class InputExampleIconClearableShorthand extends React.Component {
+  public render() {
+    return <Input icon="search" clearable placeholder="Search..." />
+  }
+}
```

Comment:
This example can be a stateless component.

Reply:
fixed